wildfirepy/net/modis/__init__.py

```diff
@@ -1 +1 @@
-from wildfirepy.net.modis.downloader import ModisBurntDownloader
+from .downloader import *
```

Comment:
Use absolute paths while importing.
```suggestion
from wildfirepy.net.modis.downloader import *
```